pkgs/applications/graphics/loupe/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, meson
+, pkg-config
+, cmake
+, gtk4
+, itstool
+, desktop-file-utils
+, ninja
+, gnome-builder
+, wrapGAppsHook4
+, libadwaita
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "loupe";
+  version = "44.alpha";
+
+  src = fetchgit {
+    url = "https://gitlab.gnome.org/BrainBlasted/loupe.git";
+    rev = "aa781d386485d76a647a7b8968b91a08e521c780";
+    sha256 = "sha256-pkHmwnK9lBjpFQbGkEjgTi5rOsYLMK6OydB/k6aMk84=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    hash = "sha256-ay2HNTcYumkwpUZkntJK4vA7puXz5swb8NwC8LG8vW8=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    cmake
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.

Reply:
hmmm, it 100% asked for `cmake` at one point but yeah now it's happy without. I guess adding `pkg-config` or `ninja` made it stop